models/dataset.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Apr 14 20:36:19 2021
+
+@author: NLEGENDN
+"""
+import os
+import numpy as np
+import torch
+from torch.utils.data import DataLoader, random_split
+import functools
+import h5py
+from PIL import Image
+CUDA = torch.cuda.is_available()
+
+if CUDA:
+    CRYO_TRAIN_VAL_DIR = os.getcwd()+"/Cryo/VAE_Cryo_V3/Data/"
```

Comment:
This is not PEP8 compliant: PEP8 recommends spaces around operators, in this case around the `+`.

To see all of your PEP8 errors, download `black`, pip install black, and use:
`black -S dataset.py`: it will reformat your code for it to be PEP8 compliant.

Reply:
thank you I'm already using black on spyder,

Reply:
Hmm that's weird that it didn't catch the missing spaces around the operator + 🤔